src/textual/app.py

```diff
@@ -399,33 +409,24 @@ def __init__(
 
     @property
     def workers(self) -> WorkerManager:
-        """The [worker](guide/workers/) manager.
-
-        Returns:
-            An object to manage workers.
-
-        """
+        """The [worker](guide/workers/) manager."""
```

Comment:
Without the `Returns:` these is nothing in the docs to link to the `WorkerManager` type.